fem/libceed/ceed.cpp

```diff
@@ -253,19 +348,65 @@ static void InitCeedTensorBasisAndRestriction(const FiniteElementSpace &fes,
                              tp_el_dof.GetData(), restr);
 }
 
+static void InitCeedTensorBasisAndRestriction(const FiniteElementSpace &fes,
+                                              const IntegrationRule &ir,
+                                              Ceed ceed, CeedBasis *basis,
+                                              CeedElemRestriction *restr)
+{
+   InitCeedTensorBasis(fes, ir, ceed, basis);
+   InitCeedTensorRestriction(fes, ir, ceed, restr);
+}
```

Comment:
You can remove it, it's a static function so it's not going to create backward compatibility issues, and it also generates warnings at compilation.